code/__DEFINES/mobs.dm

```diff
@@ -563,6 +581,7 @@ var/list/global_mutations = list() // list of hidden mutation things
 #define CASTE_IS_ROBOTIC			(1<<8)
 #define CASTE_DECAY_PROOF			(1<<9)
 #define CASTE_CAN_BE_LEADER			(1<<10)
+#define CASTE_CAN_BE_OVERWATCHED	(1<<11)
```

Comment:
Missing an identation

Reply:
Sometimes it shows up like that on git while being perfectly fine in the editor.